reports/aksversions/test_main.py

```diff
@@ -38,15 +38,12 @@ def test_main_script(mock_cosmos_client, mock_container_service_client, mock_sub
     # Call your main script function here
     main()
 
-    # Assert that the mocks were called as expected
-    mock_default_credential.assert_called_once()
```

Comment:
Can we replace this with `assert_called()`, its now called more than once but we should still check it is being called at least once or `assert_has_calls(calls, any_order=False)` with the total number expected.

Reply:
https://github.com/hmcts/version-reporter-services/pull/188/commits/a6a97f56a3d3ada7655cf8fe998528d8e72e4783